docs/060-breaking-changes.rst

```diff
@@ -98,6 +98,46 @@ or at least were more difficult to achieve prior to Solidity 0.6.0.
    ``x`` must be of type ``address``.
 
 
+Interface Changes
+=================
+
+This section lists changes that are unrelated to the language itself, but that have an effect on the interfaces of
+the compiler. These may change the way how you use the compiler on the command line, how you use its programmable
+interface or how you analyze the output produced by it.
+
+New Error Reporter
+~~~~~~~~~~~~~~~~~~
+
+A new error reporter was introduced, which aims on producing more accessible error messages on the command line.
+It is enabled by default, but passing ``--old-reporter`` falls back to the the deprecated old error reporter.
+
+Metadata Hash Options
+~~~~~~~~~~~~~~~~~~~~~
+
+The compiler now appends the `IPFS <https://ipfs.io/>`_ hash of the metadata file to the end of the bytecode by default
+(for details, see documentation on :doc:`contract metadata <metadata>`). Before 0.6.0, the compiler appended the
+`Swarm <https://ethersphere.github.io/swarm-home/>`_ hash by default, and in order to still support this behaviour,
+the new command line option ``--metadata-hash`` was introduced. It allows you to select the hash to be produced and
+appended, by passing either ``ipfs``, ``swarm`` or ``none`` as value to the ``--metadata-hash`` command line option.
```

Comment:
Please also explain that `none` can be used to completely remove it.

Reply:
Really? Too bad 😢